style.css

```diff
@@ -0,0 +1,116 @@
+.menu > div {
+    display: flex;
+    width: 35px;
+    height: 5px;
+    background-color: black;
+    margin: 6px 0;
+    position: relative;
+    
+
+  }
+  .button {
+    border:2px;
+    width: fit-content;
+    background-color: grey;
+    padding: 14px 28px;
+    font-size: 16px;
+    cursor: pointer;
+    display: inline-block;
+    color: white;
+    border-radius: 30px;
+  }
+  .button:hover {background: rgb(156, 149, 149);
+}
+
+  @media (min-width: 500px) {
+    .topnav {
+        display: flex;
+        flex-direction: row;
+        align-content: center;
+        justify-content:space-between;
+        margin: 10px;
+    }
+  }
+
+.launching-soon {
+      color: rgb(212, 29, 182);
+      letter-spacing: 2px;
+      
+  }
+.head > h1 {
+      font-size: 50px;
+      font-weight: 900px;
+      font-family:'inter', sans-serif;
+  }
+.head {
+      display: flex;
+              align-content: center;
+        justify-content:space-between;
+
+
+  }
+  .heading-left {
+      display: flex;
+      flex-direction: column;
+      width: 300px;
+     
+  }
+  .heading-right {
+      display: flex;
+      justify-content: space-around;
+      max-width: 120px;
+
+  }
+.background {
+      display: flex;
+      position: relative;
+      background-image: url(background.svg);
+      width: 1280px;
+      height: 320px; 
+      top: 40px;
+      left: 40px;
+      flex-grow: 1;
+      justify-items: center;
+  }
+.background-text {
+      position: absolute;
+      text-align: left;
+      bottom: 40px;
+      left: 40px;
+      font-size:large ;
+  }
```

Comment:
Pernelyg sudėtingas būdas padėti tekstą kampe ir containerį atitraukti nuo krašto